kong/pdk/log.lua

```diff
@@ -791,72 +804,56 @@ do
           upstream_uri = upstream_uri .. "?" .. (var.args or "")
         end
       end
-
-      return edit_result(ctx, {
+      local root = {
         request = {
           uri = request_uri,
           url = var.scheme .. "://" .. var.host .. ":" .. host_port .. request_uri,
           querystring = okong.request.get_query(), -- parameters, as a table
           method = okong.request.get_method(), -- http method
           headers = okong.request.get_headers(),
           size = request_size,
-          tls = request_tls
+          tls = build_tls_info(var),
         },
         upstream_uri = upstream_uri,
         response = {
           status = ongx.status,
           headers = ongx.resp.get_headers(),
           size = response_size,
         },
-        tries = (ctx.balancer_data or {}).tries,
         latencies = {
           kong = (ctx.KONG_PROXY_LATENCY or ctx.KONG_RESPONSE_LATENCY or 0) +
-                 (ctx.KONG_RECEIVE_TIME or 0),
+          (ctx.KONG_RECEIVE_TIME or 0),
           proxy = ctx.KONG_WAITING_TIME or -1,
           request = var.request_time * 1000
```

Comment:
these three line should be aligned  like 
```
kong = (ctx ...
             (ctx.KONG_RECEIVE_TIME or 0),
             proxy = ctx.KONG_WAITING_TIME or -1,
```

Reply:
updated